### PR TITLE
Move script templates dir, replace stock template, fix override in Skeleton, and other misc. cleanup

### PR DESCRIPTION
**Pull request by SuscipiamSingularitatem**
script templates dir is #72

### CODE REVIEW

File: ui/Console.cs [hunk 1]
```diff
@@ -95,10 +95,10 @@ public class Console : WindowDialog
...
 		} else if (commandString.StartsWith("help")) {
 			PrintToConsole(
-				"sv_cheats {integer; 1,0}: Toggles cheats\n"
-					+ "noclip: toggle noclip\n"
-					+ "fullscreen: toggle fullscreen for this session\n"
-					+ "help: this text",
+				@"sv_cheats {integer; 1,0}: Toggles cheats
+noclip: toggle noclip
+fullscreen: toggle fullscreen for this session
+help: this text",
```
Comment (HazmatDrone) [inline ui/Console.cs:101]: But why

**Review by HazmatDrone — APPROVED** [review]

File: ui/Console.cs (hunk 1 above)
Comment (Krutonium) [reply ui/Console.cs:101]: Doesn't matter any more; File should be excluded.

Comment (Krutonium) [reply ui/Console.cs:101]: Why even this change, anyway? Regardless, this file has been replaced in a different PR, so this should either but updated to the new file or excluded.

**Review by HazmatDrone — CHANGES_REQUESTED** [review]
Please drop changes to `ui/Console.cs`

**Comment by Krutonium** [comment]
I added an editorconfig based on the coding style seen in this PR. It will of course need tweaking, but it should help enforce the code style.

File: ui/Console.cs [hunk 2]
```diff
@@ -51,51 +50,40 @@ public class Console : WindowDialog
...
-				}
-				else
-				{
-					if (Player is null)
-					{
+				} else {
+					if (Player is null) {
 						PrintToConsole("Can't find the player!", true);
 						break;
 					}
```
Comment (HazmatDrone) [inline ui/Console.cs:89]:
```suggestion
				} else  if (Player is null) {
					PrintToConsole("Can't find the player!", true);
					break;
				}
```

**Comment by HazmatDrone** [comment]
@Krutonium can review and merge this if he's happy with it

**Comment by Krutonium** [comment]
I see nothing that needs changes, OK to merge.

**Review by Krutonium — APPROVED** [review]
Looks  Good

**Pull request merged**